Sources/SwiftUIBackports/iOS/UIHostingConfiguration/Cells/UICollectionViewCell.swift

```diff
@@ -2,25 +2,31 @@ import SwiftUI
 import ObjectiveC
 
 #if os(iOS) || os(tvOS)
+
+extension UICollectionViewCell {
+    
+    private static var configuredViewAssociatedKey: Void?
+    
+    fileprivate var configuredView: UIView? {
+        get { objc_getAssociatedObject(self, &Self.configuredViewAssociatedKey) as? UIView }
```

Comment:
While I’m not averse to having a dedicated key, I’m sure I’ve used `#function` over the years without issue. I can see why the issue occurred on `Backport` given its a value-type. But it should be fine here and would remove the need for a secondary definition?

Reply:
I tested the `#function` but i doesn't work for me,  😵‍💫 Am I missing something?
```
class Value: NSObject {
    
    var c = 0
    
    init(c: Int = 0) {
        self.c = c
    }
}

class AClass: NSObject {
    
    var value: Value? {
        get {
            return objc_getAssociatedObject(self, #function) as? Value
        }
        set {
            objc_setAssociatedObject(self, #function, newValue, .OBJC_ASSOCIATION_RETAIN)
        }
    }
}

let object = AClass()
object.value = Value(c: 3)
print(object.value) // nil

```

Reply:
Interesting. I'm fine with your implementation 👍👍

Thank you for confirming 👍